src/cogent3/core/new_alignment.py

```diff
@@ -199,7 +208,10 @@ def to_rich_dict(self):
         else:
             start, stop = self.start, self.stop
 
```

Comment:
line 213 changes `.seq` from a `SeqData` instance, to a `str`. My instinct is that it should be a new `SeqData` instance where the seq corresponding to `seqid` is sliced